llvm/tools/llvm-exegesis/lib/Target.h

```diff
@@ -262,7 +262,8 @@ class ExegesisTarget {
       Benchmark::ModeE Mode, const LLVMState &State,
       BenchmarkPhaseSelectorE BenchmarkPhaseSelector,
       BenchmarkRunner::ExecutionModeE ExecutionMode,
-      Benchmark::ResultAggregationModeE ResultAggMode = Benchmark::Min) const;
+      Benchmark::ResultAggregationModeE ResultAggMode = Benchmark::Min,
+      unsigned BenchmarkRepeatCount = 30) const;
```

Comment:
Do we need the default value now that we have the flag in main ?

Reply:
Shouldn't be needed. I've removed it.

Also caught that I never plumbed the flag into the call in `llvm-exegesis.cpp`, so I've also added that in this patch.